libraries/apollo-runtime/build.gradle.kts

```diff
@@ -89,6 +88,13 @@ kotlin {
       }
     }
 
+    findByName("linuxMain")?.apply {
+      dependencies {
+        implementation(libs.kotlinx.io.okio)
+        implementation(libs.ktor.client.curl)
```

Comment:
We usually avoid ktor in the main artifacts (ktor support is provided in a separate artifact [there](https://github.com/apollographql/apollo-kotlin-ktor-support)). But as long as it doesn't leak to other platforms and it's allowing to unblock linux, LGTM

Reply:
Understood. This was a question to @BoD at the start of this in the issue thread if ktor would be a reasonable solution for the platform now. If there is another alternative I'm open to doing some research to evaluate it.

Reply:
Sorry I forgot the initial context. The alternative is calling the [libcurl C API](https://curl.se/libcurl/c/) but this is probably going to be more involved so agree with @BoD that ktor is a reasonable solution for now.